cmake/protobuf-config.cmake.in

```diff
@@ -9,12 +9,15 @@ include("${CMAKE_CURRENT_LIST_DIR}/protobuf-targets.cmake")
 
 function(protobuf_generate)
   include(CMakeParseArguments)
-  set(_singleargs LANGUAGE OUT_VAR)
+
+  set(_options APPEND_PATH)
+  set(_singleargs LANGUAGE OUT_VAR EXPORT_MACRO)
   if(COMMAND target_sources)
     list(APPEND _singleargs TARGET)
   endif()
+  set(_multiargs PROTOS IMPORT_DIRS GENERATE_EXTENSIONS)
 
-  cmake_parse_arguments(protobuf_generate "APPEND_PATH" "${_singleargs}" "PROTOS;IMPORT_DIRS;GENERATE_EXTENSIONS" "${ARGN}")
+  cmake_parse_arguments(protobuf_generate "${_options}" "${_singleargs}" "${_multiargs}" "${ARGN}")
```

Comment:
It seems they corrected whatever problem existed here. I merged the upstream change.